### DISCUSSION TITLE
plate recognition for moving cars

### DISCUSSION THREAD

**Discussion by belmont** [robmarkcole/HASS-plate-recognizer#45] [Ideas]
any idea how to make it work with moving cars?
the object detector of Frigate detects the car earlier than the license plate would be clearly visible.
I use the Frigate object detector as binary sensor to trigger the license plate recognizer.
I feed the frigate camera to the plate recognizer but it never picks a photo that is close enough to recognize the number plate.

Basically I am trying to use 2 object detector, one for recognizing cars (done by Frigate) and another to recognize the number plate (using this)

I use "only" full HD resolution on my Hikvision cameras and this is good enough to read out license plates for my human eyes. I know about LPR cameras but they are expensive. They starts from 700USD. I guess these cams have a local object detector which triggers on car and then the camera zoom on the license plate object.

**Comment by totalitarian** [comment]
Ever get an answer to this?

**Comment by robmarkcole** [comment]
The HD image is possibly downsized by plate recognizer. To prevent this the integration would need to be updated to slice the image and process the slices (at HD). I expect the $700 cameras are capturing at high FPS to minimise motion blur

**Comment by totalitarian** [comment]
Somehow we need to get frigate to pass a clean image to plate recogniser....

**Reply by robmarkcole** [reply]
I'm not familiar with the processing that frigate does, but assume it is documented somewhere
